structuredheadings/plugin.js

```diff
@@ -65,6 +66,11 @@
         });
       };
 
+      //some friendly setup for level changes
+      var headerList = Object.keys(allowedElements);
```

Comment:
may be worth leaving a comment that this doesn't work in IE8.  Though few customers still are using it + we don't officially support.